source/i18n/en-US.ts

```diff
@@ -1,4 +1,75 @@
+    sina_weibo: 'Sina Weibo',
+    wechat_public_account: 'WeChat official account',
+    proudly_developed_with: 'Proudly developed with',
+    development: 'development',
+    back_to_top: 'Back To Top',
+
+    chengdu_web_full_stack_conference_2019:
+        '2019 Chengdu Web Full Stack Conference',
+    coscon22_chengdu_branch_venue:
+        "COSCon'22 (China Open Source Conference) Chengdu Branch Venue",
+    coscon21_main_venue: "COSCon'21 (China Open Source Conference) Main Venue",
+    coscon20_chengdu_branch_venue:
+        "COSCon'20 (China Open Source Conference) Chengdu Branch Venue",
+    chengdu_web_frontend_conference_2018:
+        '2018 Chengdu Web Frontend Conference',
+    chongqing_web_frontend_conference_2018:
+        '2018 Chongqing Web Frontend Conference',
+    chengdu_react_tech_conference_2018: '2018 Chengdu React Tech Conference',
+    xinyao_cup_code_for_city_hackathon: 'Xinyao Cup Code for City Hackathon',
+    chengdu_web_frontend_conference_2017:
+        '2017 Chengdu Web Frontend Conference',
+    adhering_to_freecodecamp_international_community_spirit:
+        'Adhering to freeCodeCamp International Community Spirit',
+
+    freecodecamp_global_largest_open_source_learning_platform:
+        "freeCodeCamp.org, the world's largest open-source programming learning platform (Top 10 on GitHub), aims to become the Wikipedia of computer education",
+    leading_freecodecamp_chinese_community_trend:
+        'Leading the trend of freeCodeCamp Chinese Community',
+    first_to_establish_daily_activities_guidelines:
+        'The first to establish daily activity guidelines, design city community logos and mascots, host hackathons and tech conferences, and experiment with open-source offline operations',
+    chengdu_most_active_programmer_community:
+        "Chengdu's Most Active Programmer Community",
+    numerous_chinese_tech_community_allies:
+        'Numerous Chinese Tech Community Allies',
+    allies_summary:
+        'Open Source Society, Open Source Workshop, Microsoft MVP Community, Chengdu Google Developer Community, Alibaba Cloud ACE Chengdu Local Group, Chengdu Linux User Group, Chongqing Frontend Community...',
+    more_exciting_awaits_your_participation:
+        'More exciting awaits your participation...',
+    years_continue_to_undertake: ({
+        which_year,
+        number_of_times,
+        long_sentence
+    }: Record<'which_year' | 'number_of_times' | 'long_sentence', string>) =>
+        textJoin(
+            which_year,
+            'years continue to undertake',
+            number_of_times, // Ensure it's a string for textJoin
+            long_sentence
+        ),
+
```

Comment:
```suggestion
            number_of_times,
            long_sentence
        ),
```
此处无需变更，类型都是写对的，只要注意外面使用的时候按照类型来传值就行了。